python_modules/dagit/dagit/webapp/src/plugins/ipynb.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { Button, Classes, Dialog } from "@blueprintjs/core";
+import { IPluginInterface, IPluginSidebarProps } from ".";
+
+// TODO: Replace REACT_APP_GRAPHQL_URI with "DAGIT_SERVER_URI" without path
+const NOTEBOOK_RENDERER_URI = process.env.REACT_APP_GRAPHQL_URI
+  ? process.env.REACT_APP_GRAPHQL_URI.replace("/graphql", "/notebook")
+  : "/notebook";
+
+export class SidebarComponent extends React.Component<IPluginSidebarProps> {
```

Comment:
[nit] Should it be NotebookComponent or smth like that?

```suggestion
export class NotebookComponent extends React.Component<IPluginSidebarProps> {
```